src/Components/CreateImageWizard/ValidatedInput.tsx

```diff
@@ -168,6 +169,7 @@ export const ValidatedInput = ({
         aria-label={ariaLabel || ''}
         onBlur={handleBlur}
         placeholder={placeholder || ''}
+        {...props}
```

Comment:
Here also - it would be nicer to spread the props so we have more control. But it works and if we open an issue to look at this in a follow up I'll be happy.